### PR TITLE
arm64 keywords

### PR DESCRIPTION
**Pull request by stikonas**
sys-fs/dfc
mail-filter/dspam
app-portage/euses
sys-kernel/dracut
app-misc/mc
games-roguelike/nethack
mail-filter/libmilter
app-admin/spectre-meltdown-checker
app-portage/ufed

### CODE REVIEW

File: app-misc/mc/mc-4.8.20-r2.ebuild [hunk 1]
```diff
@@ -13,7 +13,7 @@ SRC_URI="http://ftp.midnight-commander.org/${MY_P}.tar.xz"
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x86-solaris"
```
Comment (hsoft) [inline app-misc/mc/mc-4.8.20-r2.ebuild]: removing ~arm?

Comment (stikonas) [reply app-misc/mc/mc-4.8.20-r2.ebuild]: argh, my bad :( Sorry for that. Will push a fix

Comment (hsoft) [reply app-misc/mc/mc-4.8.20-r2.ebuild]: use `ekeyword` to modify an ebuild's `KEYWORDS` to avoid those embarrassing mistakes.

Comment (stikonas) [reply app-misc/mc/mc-4.8.20-r2.ebuild]: Okay thanks. This is my first pull request to main portage tree, so obviously I need to learn some things.

**Comment by steev** [comment]
Would you mind updating the dfc keywords now that it has been stabilized on amd64/x86?

**Comment by steev** [comment]
👍

**Comment by leio** [comment]
Have you actually tested all these revisions and versions of the same package, or really just the latest? Probably only the latest should be keyworded, especially if only that is tested..

**Comment by stikonas** [comment]
On Wed, Nov 07, 2018 at 02:32:09AM -0800, Mart Raudsepp wrote:
> Have you actually tested all these revisions and versions of the same package, or really just the latest? Probably only the latest should be keyworded, especially if only that is tested..

I tested all revisions except for -9999 which is a moving target anyway. But I am fine with just the latest if you wish so. Let me know and can update pull request.
(those older revisions were stable on some arch).


Also, what should I do with sys-kernel/dracut-048-r1? It mostly works but there is an issue with cryptsetup. I reported an upstream bug
https://github.com/dracutdevs/dracut/issues/473

By the way, should I mark all previous versions of dracut -arm64. They is known not to work.

**Comment by leio** [comment]
No -arm64 marking; no keywords ought to be sufficient for that. -arch is mostly used if it's known to not work due to architectural reasons - like no binaries for it in case of binary packages (then it's usually KEYWORDS="-* amd64" or so), or no required kernel support for that arch, or such.

If you tested them all, then I guess I can include all (and only test the latest myself); 9999 changes has no effect - they don't have keywords; but I guess by adding it there to the unused conditional branch, then we get to retain the ~arm64 keyword if maintainer copies bumps from 9999 and doesn't compare KEYWORDS properly with the previous version.

**Comment by leio** [comment]
I'll skip dracut for not then, so we get the rest in, as it seems I'd have to look closer into the topic, which would unnecessary delay keywords for the rest. Please handle that in your this weeks PR or something :)
I can just drop that commit locally, no need to rebase for that

**Comment by stikonas** [comment]
@leio 
```
sys-fs/dfc-3.1.1
games-roguelike/nethack-3.6.1
app-portage/euses/euses-2.5.9
mail-filter/dspam/dspam-3.10.2-r1
app-portage/ufed-0.92
app-portage/ufed-9999
mail-filter/libmilter-1.0.2
sys-kernel/dracut-048-r1
sys-kernel/dracut-9999
```
Last one (i.e. dracut) has a small issue, so let me know if it is suitable for keywording
https://github.com/dracutdevs/dracut/issues/473

**Comment by stikonas** [comment]
There were some dracut fixes that might be useful to arm64

A bit later I'll update my pull request to
```
sys-fs/dfc-3.1.1
games-roguelike/nethack-3.6.1
app-portage/euses/euses-2.5.9
mail-filter/dspam/dspam-3.10.2-r1
app-portage/ufed-0.92
app-portage/ufed-9999
mail-filter/libmilter-1.0.2
sys-kernel/dracut-049-r1
sys-kernel/dracut-9999
```

**Comment by flappyports** [comment]
@stikonas, are you still interested in this PR?  If so, please fix-up the conflicts.

**Comment by stikonas** [comment]
I'll update this pull request, It's like the third time I have to solve merge conflict in these 5 months but oh well...

(And signoff that dfc commit, not sure why that one was missed out).

**Comment by stikonas** [comment]
> Please sign-off on this commit

Done

**Pull request closed without merging**